config/database.yml.sample

```
@@ -1,12 +1,27 @@
 ---
-development: &local
-  adapter: trilogy
-  host: 127.0.0.1
-  port: 3306
-  database: lobsters_dev
-  username: root
-  password: localdev
+development:
+  primary: &local
+    adapter: trilogy
```

```suggestion
    adapter: trilogy
    encoding: utf8mb4
```

Not sure why this isn't present already, I'd rather make it a suggestion here than give you a merge conflict.